client/components/Profile.jsx

```diff
@@ -6,63 +6,87 @@ import Pro from './Pro.jsx';
+            last_name: '',
+            full_name: '',
+            email: '',
+            location: '',
+            date_joined: '',
+            favorite_activities: [],
+            bio: '',
+            photo_url: '',
+            pro: false
+            // Could still add Reputation and followers
+        };
 
-    this.convertDate = this.convertDate.bind(this);
-  }
+        this.convertDate = this.convertDate.bind(this);
+    }
 
     convertDate() {
         const monthNames = ["January", "February", "March",
             "April", "May", "June", "July", "August", "September",
             "October", "November", "December"];
-        let date = new Date(this.state.date_joined +'Z');
+        let date = new Date(this.state.date_joined + 'Z');
 
         return monthNames[date.getMonth() - 1] + ' ' + date.getFullYear().toString();
     }
 
+    componentDidMount() {
+
+        fetch('http://localhost:3002/user/' + window.location.pathname.substring(1))
+            .then((res) => {
+                return res.json();
+            })
+            .then((res) => {
+                console.log(res.data);
+                this.setState({
+                    id: res.id,
+                    first_name: res.data.attributes.first_name,
+                    last_name: res.data.attributes.last_name,
+                    full_name: res.data.attributes.first_name + res.data.attributes.last_name,
+                    email: res.data.attributes.email,
+                    location: res.data.attributes.location,
+                    date_joined: res.data.attributes.date_joined,
+                    favorite_activities: res.data.attributes.favorite_activities,
+                    bio: res.data.attributes.bio,
+                    photo_url: res.data.attributes.photo_url,
+                    pro: res.data.attributes.pro
+                })
+            });
+    }
+
     render() {
         return (
-        <div id="profile-container">
-            {/* Possibly Add navigation topbar for other profile components */}
-            <div id="profile-subbar">
-                <h3>Profile</h3>
-            </div>
-            <div id="profile-main">
-                <div id="profile-info">
-                    <div id="profile-left-content">
-                        <div id="profile-image-container">
-                            <img id="profile-image" src={this.state.photo_url} alt={this.state.full_name}/>
+            <div id="profile-container">
+                {/* Possibly Add navigation topbar for other profile components */}
+                <div id="profile-subbar">
+                    <h3>Profile</h3>
+                </div>
+                <div id="profile-main">
+                    <div id="profile-info">
+                        <div id="profile-left-content">
+                            <div id="profile-image-container">
+                                <img id="profile-image" src={this.state.photo_url} alt={this.state.full_name} />
```

Comment:
something is wrong with photo_url with dynamic routes

![image](https://user-images.githubusercontent.com/4239492/50048144-80980500-0079-11e9-9a48-10f77be29a0a.png)

Reply:
acruallt hmm its something to do with this lorempixel site, seems unstable a lil bit,  ideally you would have 100 images on s3 call it a day, as every refresh results in different images. but ill resolve this comment for now.